app/Http/Controllers/Api/V1/ActivityController.php

```diff
@@ -74,9 +75,25 @@ public function store(Request $request)
      * @return Response
      */
     public function show(Activity $activity)
-    {
+    {        
+        $data = ['h5p_parameters' =>  null,  'user_name' => null,  'user_id' => null];   
+        
+        if( $activity->playlist->project->user ){
```

Comment:
I mean add space before "(" and after ")".

Reply:
my bad. added!